src/main/java/StringCalculator/domain/Operators.java

```diff
@@ -0,0 +1,27 @@
+package StringCalculator.domain;
+
+import StringCalculator.constant.Operator;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class Operators {
+
+    private final Queue<Operator> operators;
+
+    public Operators() {
+        operators = new LinkedList<>();
+    }
+
+    public Operators(Queue<Operator> operators) {
+        this.operators = operators;
+    }
+
+    public boolean hasNext() {
+        return 0 < operators.size();
```

Comment:
`isEmpty()` 메서드를 활용하여 다음처럼 작성해볼 수도 있을 것 같습니다.
```suggestion
        return !operators.isEmpty();
```